composer.json

```diff
@@ -10,7 +10,8 @@
         "webmozart/glob": "^4.3"
     },
     "conflict": {
-        "api-platform/core": "^2.6"
+        "api-platform/core": "^2.6",
```

Comment:
```suggestion
```
Can we remove this conflict? Is still necessary?

Reply:
Yes, but I'll remove it in the PR for Sylius 1.10 and PHP 8.0 (issue here #7)